src/locales/en.json

```diff
@@ -103,7 +103,11 @@
 			"login_placeholder": "CAS-UTC or email",
 			"password_label": "Password :",
 			"password_placeholder": "Connection password",
-			"bad_login_password": "Wrong identifiants",
+			"bad_login_password": "Wrong credentials",
+			"i_am_ext": "I am extern",
+			"i_am_cas": "I am CAS-UTC",
```

Comment:
"Try as an extern..."
"Try with CAS-UTC..."

Reply:
Actually, you can't try as something, the app auto detects what kind of type the user is. So he can be one of these types but not try as a type if it is not of the type